### PR TITLE
Ember 3.1 and optional features

### PR DESCRIPTION
**Pull request by rwwagner90**

### CODE REVIEW

File: app/templates/application.hbs [hunk 1]
```diff
@@ -1,9 +1,11 @@
-<main-navbar></main-navbar>
+<div class="ember-view">
```
Comment (locks) [inline app/templates/application.hbs]: I think we can safely remove this `div`, can you try?

Comment (rwwagner90) [reply app/templates/application.hbs]: Yep, seems to work 👍

File: package.json [hunk 2]
```diff
@@ -35,12 +36,12 @@
     "ember-cli-sri": "^2.1.0",
     "ember-cli-uglify": "^2.0.0",
     "ember-cli-yaml-module": "^1.0.0",
-    "ember-data": "~3.0.0",
```
Comment (locks) [inline package.json]: Can you remove this dependency?

**Comment by locks** [comment]
We're not deploying to Netlify, nevermind :P

**Pull request merged**